tests/testenv-container/make-test.sh

```diff
@@ -8,4 +8,14 @@ cp /krb5.conf /etc/krb5.conf
 cat /resolv.conf > /etc/resolv.conf
 chown -R builder2:builder2 /app/
 echo 'password145Qw!' | kinit administrator@DOMAIN.ALT || :
-cd /app/ && rm -rf $BUILD_DIR && mkdir $BUILD_DIR && cd $BUILD_DIR && pwd && cmake -DLIBDOMAIN_BUILD_TESTS:BOOL=ON -B . .. && make && ctest --verbose
+cd /app/ && rm -rf $BUILD_DIR && mkdir $BUILD_DIR && cd $BUILD_DIR && pwd 
+
+if [[ "$PIPELINE_TYPE" == "MEMCHECK" ]]; then
+  cmake -DLIBDOMAIN_BUILD_TESTS:BOOL=ON -DCMAKE_BUILD_TYPE=Debug -B . .. && \
+    make && \
+    ctest --overwrite MemoryCheckCommandOptions="-q --leak-check=full --error-exitcode=99" -T memcheck --verbose
```

Comment:
For consistency, please use either 2 or 4 spaces for text indentation, but not a mix of both.